mock/channels/channelService.js

```diff
@@ -1,40 +1,138 @@
-class AppChannelService {
-    async joinChannel() {
-        return await window.fdc3.getOrCreateChannel("test-channel");
-    }
+class Fdc3CommandExecutor {
+  //execute commands in order
+  async executeCommands(orderedCommands, config, testId) {
```

Comment:
`testId` is in config so should be  `async executeCommands(orderedCommands, config)`